libs/user_extensions/include/TTAlpsSelections.hpp

```diff
@@ -11,19 +11,27 @@
 #include "EventProcessor.hpp"
 #include "Helpers.hpp"
 #include "PhysicsObject.hpp"
+#include "TTAlpsDimuonSelections.hpp"
 
 class TTAlpsSelections {
  public:
   TTAlpsSelections();
   
-  // Selections targetting semi-leptonic ttbar + two muons. Requires:
+  // Selections targetting SR semi-leptonic ttbar + two muons. Requires:
   // - 1 good e/μ (the top-lepton)
   // - at least 2 additional good muons (different than the top-lepton)
   // - at least 4 good jets
   // - at least 1 good b-tagged jet
   // - some amount of MET
+  // - at least one good dimuon pair with mass excluding the JPsi window
   bool PassesSignalLikeSelections(const std::shared_ptr<Event> event, std::shared_ptr<CutFlowManager> cutFlowManager);
   void RegisterSignalLikeSelections(std::shared_ptr<CutFlowManager> cutFlowManager);
+  
+  void RegisterInitialDimuonCuts(std::shared_ptr<CutFlowManager> cutFlowManager);
+  void RegisterDimuonSelections(std::shared_ptr<CutFlowManager> cutFlowManager);
+  bool PassesDimuonSelections(const std::shared_ptr<Event> event, std::shared_ptr<CutFlowManager> cutFlowManager);
```

Comment:
Hah it's actually because I'm following your naming conventions. First one I use RegisterCut, the others follows similar naming as you did for RegisterSingleLeptonSelections and PassesSingleLeptonSelections. I can change RegisterInitialDimuonCuts but it's not really selections I just check if "initial" cut exist, and if not I do RegisterCut("initial");

Reply:
I don't know that is a very big change to make everywhere. I have TTAlpsSelections and TTAlpsDimuonSelections with selections, and we have CutFlowManager with cuts. I prefer TTAlpsSelections but it makes sense to call it cuts for the cut flow